Formula/kryptor.rb

```diff
@@ -0,0 +1,30 @@
+class Kryptor < Formula
+  desc "Simple, modern, and secure file encryption and signing"
+  homepage "https://www.kryptor.co.uk/"
+  url "https://github.com/samuel-lucas6/Kryptor/archive/v3.1.0.tar.gz"
+  sha256 "e7372ac4ccc4676fa6079bcc7be257c3677003b8ec33df486a4d43ba8bd1f3af"
+  license "GPL-3.0-or-later"
+
+  depends_on "dotnet"
+
+  def install
+    os = OS.mac? ? "osx" : OS.kernel_name.downcase
+    arch = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch.to_s
+
+    system "dotnet", "publish", "src/KryptorCLI/KryptorCLI.csproj",
+           "-c", "Release",
+           "-r", "#{os}-#{arch}",
+           "-p:PublishTrimmed=true",
+           "-p:IncludeNativeLibrariesForSelfExtract=true",
+           "-p:PublishSingleFile=true",
+           "--self-contained", "true"
```

Comment:
Do you need to package .NET libs/runtime?

Most of our .NET formulae are [framework-dependent](https://docs.microsoft.com/en-us/dotnet/core/deploying/#publish-framework-dependent) and just keep a `dotnet` dependency. We do dependent-testing when .NET is updated, so it should hopefully avoid any issue with incompatible runtime while providing advantage of latest security patches.

Reply:
That's a fair point. I only did it this way because that's how I build the binaries for the ZIP releases and other packages I'm maintaining for Windows.